tests/test_api.py

```diff
@@ -106,7 +106,7 @@ def get_MockComponent_attribute():
     data_model_grapher = DataModelGraph(parsed_data_model)
 
     # Generate graph
-    graph_data_model = data_model_grapher.generate_data_model_graph()
+    graph_data_model = data_model_grapher.graph
```

Comment:
@mialy-defelice Here I already made a change for `get_MockComponent_attribute()` function. Does this what you meant in your comment?

Reply:
Yes, thank you!